packages/now-python/test/fixtures/00-request-path/vercel.json

```diff
@@ -0,0 +1,59 @@
+{
+  "version": 2,
+  "builds": [{ "src": "api/*.py", "use": "@vercel/python" }],
+  "probes": [
```

Comment:
These probes are the _fixed_ versions from the two removed fixtures in this PR (`10-` and `13-`).